### PR TITLE
Fix all references to "MoveIt" to include exclamation mark e.g. "MoveIt!"

### PR DESCRIPTION
**Pull request by MohmadAyman**
### Description

Fix all references to "MoveIt" to include exclamation mark e.g. "MoveIt!".

### Checklist
- [ ] Optional: Decide if this should be cherry-picked to other current ROS branches (Indigo, Jade, Kinetic)

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Comment by davetcoleman** [comment]
Somehow you ran the wrong type of clang-format - it should not have caused this many changes. Can you remove the "apply clang format" commit and try again using the instructions here:

http://moveit.ros.org/documentation/contributing/code/

Maybe you have the wrong version of clang-format?

**Comment by MohmadAyman** [comment]
Yes I've noticed that `clang` doesn't usually reorder includes according to their spelling, reinstalled it and works normally now, thanks.

File: moveit_core/controller_manager/include/moveit/controller_manager/controller_manager.h [hunk 1]
```diff
@@ -153,21 +153,22 @@ MOVEIT_CLASS_FORWARD(MoveItControllerManager);
...
       structure describes that controller's state. */
   struct ControllerState
   {
     ControllerState() : active_(false), default_(false)
     {
     }
 
-    /** \brief A controller can be active or inactive. This means that MoveIt could activate the controller when needed,
+    /** \brief A controller can be active or inactive. This means that MoveIt! could activate the controller when
+       needed,
```
Comment (mcevoyandy) [inline moveit_core/controller_manager/include/moveit/controller_manager/controller_manager.h]: Fix wrapping with line below

File: moveit_core/controller_manager/include/moveit/controller_manager/controller_manager.h [hunk 2]
```diff
@@ -153,21 +153,22 @@ MOVEIT_CLASS_FORWARD(MoveItControllerManager);
...
         and de-activate controllers that overlap (control the same set of joints) */
     bool active_;
 
     /** \brief It is often the case that multiple controllers could be used to execute a motion. Marking a controller as
        default
-        makes MoveIt prefer this controller when multiple options are available. */
+        makes MoveIt! prefer this controller when multiple options are available. */
```
Comment (mcevoyandy) [inline moveit_core/controller_manager/include/moveit/controller_manager/controller_manager.h]: text wrapping

**Comment by MohmadAyman** [comment]
Fixed text wrap :+1: @mcevoyandy

**Comment by davetcoleman** [comment]
cherry-pick: https://github.com/ros-planning/moveit/pull/1030

**Pull request merged**